app/services/image_path_service.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ImagePathService
```

Comment:
We agreed we dont use Service in the services names: https://community.openfoodnetwork.org/t/how-to-name-a-new-service-in-ofn/1530/9
We dont need to be strict but we can easily follow the convention here with a service name like ImagePathBuilder.call or just ImagePath.call